src/System.IO.MemoryMappedFiles/tests/MemoryMappedViewStream/ReadWrite.cs

```diff
@@ -339,143 +339,140 @@ public bool runTest()
                 }
             }
 
-            if (Interop.PlatformDetection.OperatingSystem == Interop.OperatingSystem.Windows) // currently don't support sharing views of pagefile backed MMFs on Unix
+            // Use a pagefile backed MMF instead of file backed
+            using (MemoryMappedFile mmf = MemoryMappedFile.CreateNew(null, pageSize * 10))
```

Comment:
FYI: add `?w=1` to the github url to ignore whitespace changes (you can't comment on lines in this mode though).